src/ovirtapi/transform.js

```diff
@@ -468,6 +468,9 @@ const StorageDomain = {
       name: storageDomain.name,
       type: storageDomain.type,
 
+      availableSpace: convertInt(storageDomain.available),
```

Comment:
this appears unrelated

Reply:
So...I added those while I was reading the API thinking that those sizes would be a nice thing to show to users about a storage domain when they're creating disks.